travis/docker/Dockerfile.travis

```
@@ -31,7 +46,18 @@ USER pwntools
 WORKDIR /home/pwntools
 
 # Copy in the Doctest script
-COPY doctest /home/pwntools/doctest
+COPY doctest /home/pwntools
+COPY tmux.sh /home/pwntools
+
+# Do everything in UTF-8 mode!
+ENV LANG=C.UTF-8
+ENV LC_ALL=C.UTF-8
+
+# Install pwndbg!
+RUN git clone https://github.com/pwndbg/pwndbg.git
+WORKDIR /home/pwntools/pwndbg
+RUN ./setup.sh
 
 # Set entry point to doctest by default
+WORKDIR /home/pwntools
 ENTRYPOINT /home/pwntools/doctest
```

Maybe a trailing newline BTW

```suggestion
ENTRYPOINT /home/pwntools/doctest

```

---

EZ PZ Done